src/services/api/auth.ts

```diff
@@ -38,6 +38,24 @@ export interface ResponseUpdateUserDataType {
   updatedUser: ResponseLoginDataType;
   error?: string;
 }
+export interface ResponseForgotPasswordDataType {
+  info: string;
+
+  error?: string;
+}
```

Comment:
Дублировать не стоит
```suggestion

}
```